============================= test session starts ==============================
platform darwin -- Python 3.9.18, pytest-7.4.0, pluggy-1.0.0
rootdir: /Users/mcharkhabi/public-github/cs244b/tests
collected 0 items / 1 error

==================================== ERRORS ====================================
__________________ ERROR collecting test_scheduler_worker.py ___________________
../../../anaconda3/envs/cs244b/lib/python3.9/site-packages/_pytest/python.py:617: in _importtestmodule
    mod = import_path(self.path, mode=importmode, root=self.config.rootpath)
../../../anaconda3/envs/cs244b/lib/python3.9/site-packages/_pytest/pathlib.py:565: in import_path
    importlib.import_module(module_name)
../../../anaconda3/envs/cs244b/lib/python3.9/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1030: in _gcd_import
    ???
<frozen importlib._bootstrap>:1007: in _find_and_load
    ???
<frozen importlib._bootstrap>:986: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:680: in _load_unlocked
    ???
../../../anaconda3/envs/cs244b/lib/python3.9/site-packages/_pytest/assertion/rewrite.py:178: in exec_module
    exec(co, module.__dict__)
test_scheduler_worker.py:6: in <module>
    from scheduler_client import SchedulerClient
../scheduler_client.py:1: in <module>
    import api_pb2_grpc
../api_pb2_grpc.py:5: in <module>
    import api_pb2 as api__pb2
E     File "/Users/mcharkhabi/public-github/cs244b/tests/../api_pb2.py", line 528
E       ],
E       ^
E   SyntaxError: closing parenthesis ']' does not match opening parenthesis '(' on line 468
=========================== short test summary info ============================
ERROR test_scheduler_worker.py
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
=============================== 1 error in 0.11s ===============================
